docs/static/releasenotes.asciidoc

```diff
@@ -69,6 +70,34 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-15-5]]
+=== Logstash 8.15.5 Release Notes
+
+[[notable-8-15-5]]
+==== Notable issues fixed
+
+* PipelineBusV2 deadlock proofing: We fixed an issue that could cause a deadlock when the pipeline-to-pipeline feature was in use, causing pipelines (and consequently) {ls} to never terminate https://github.com/elastic/logstash/pull/16681[#16681]
+* We reverted a change in BufferedTokenizer (https://github.com/elastic/logstash/pull/16482[#16482]) that improved handling of large messages but introduced a double encoding bug https://github.com/elastic/logstash/pull/16687[#16687].
```

Comment:
@jsvd is there any reason why this revert change wasn't included in [8.16.1 release note](https://github.com/elastic/logstash/pull/16691/files)? - I am assuming release note generator catched up this.

Reply:
Looks like we [added a change](https://github.com/elastic/logstash/pull/16482) and [reverted back](https://github.com/elastic/logstash/pull/16688) in 8.16.1?

Reply:
The reason this wasn't added to the 8.16.1 release notes is that the initial fix only landed on 8.16 after 8.16.0 GA, so both the "bug" and the "fix" where added within 8.16.1, essentially a no-op.
While in 8.15 the issue was introduced in 8.15.4, and the revert was added to 8.15.5.